.github/CONTRIBUTING.md

```diff
@@ -236,220 +243,219 @@ First, read the comments in [this BYOND thread](http://www.byond.com/forum/?post
+```DM
+//Bad
 /datum/proc/do_the_thing(thing_to_do)
-  switch(thing_to_do)
-    if(1)
-      (...)
-    if(2)
-      (...)
-````
+    switch(thing_to_do)
+        if(1)
+            do_stuff()
+        if(2)
+            do_other_stuff()
+```
 There's no indication of what "1" and "2" mean! Instead, you should do something like this:
-````DM
+```DM
+//Good
 #define DO_THE_THING_REALLY_HARD 1
 #define DO_THE_THING_EFFICIENTLY 2
+
 /datum/proc/do_the_thing(thing_to_do)
-  switch(thing_to_do)
-    if(DO_THE_THING_REALLY_HARD)
-      (...)
-    if(DO_THE_THING_EFFICIENTLY)
-      (...)
-````
+    switch(thing_to_do)
+        if(DO_THE_THING_REALLY_HARD)
+            do_stuff()
+        if(DO_THE_THING_EFFICIENTLY)
+            do_other_stuff()
+```
 This is clearer and enhances readability of your code! Get used to doing it!
 
 ### Control statements
 (if, while, for, etc)
 
-* All control statements must not contain code on the same line as the statement (`if(condition) return`)
 * All control statements comparing a variable to a number should use the formula of `thing` `operator` `number`, not the reverse
   (eg: `if(count <= 10)` not `if(10 >= count)`)
 * All control statements must be spaced as `if()`, with the brackets touching the keyword.
-* Do not use one-line control statements.
-  Instead of doing
-  ```
+* All control statements must not contain code on the same line as the statement.
+  ```DM
+  //Bad
   if(x) return
-  ```
-  You should do
-  ```
+
+  //Good
   if(x)
-    return
+      return
   ```
 
 ### Player Output
-Due to the use of "Goonchat", Paradise requires a special syntax for outputting text messages to players. Instead of `mob/client/world << "message"`,
-you must use `to_chat(mob/client/world, "message")`. Failure to do so will lead to your code not working.
+Due to the use of "Goonchat", Paradise requires a special syntax for outputting text messages to players. Instead of `mob << "message"`,
+you must use `to_chat(mob, "message")`. Failure to do so will lead to your code not working.
 
 ### Use early return
 Do not enclose a proc in an if-block when returning on a condition is more feasible.
 
 This is bad:
 ````DM
 /datum/datum1/proc/proc1()
-  if(thing1)
-    if(!thing2)
-      if(thing3 == 30)
-        do stuff
+    if(thing1)
+        if(!thing2)
+            if(thing3 == 30)
+                do stuff
 ````
 This is good:
 ````DM
 /datum/datum1/proc/proc1()
-  if(!thing1)
-    return
-  if(thing2)
-    return
-  if(thing3 != 30)
-    return
-  do stuff
+    if(!thing1)
+        return
+    if(thing2)
+        return
+    if(thing3 != 30)
+        return
+    do stuff
 ````
 This prevents nesting levels from getting deeper then they need to be.
 
-### Uses addtimer() instead of sleep() or spawn()
-If you need to call a proc after a set amount of time, use addtimer() instead of spawn() / sleep() where feasible.
+### Uses `addtimer()` instead of `sleep()` or `spawn()`
```

Comment:
```suggestion
### Use `addtimer()` instead of `sleep()` or `spawn()`
```

Reply:
is uses `addtimer()` instead of `sleep()` or `spawn()` yaya 🐦